contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -51,7 +57,20 @@ abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHand
   }
 
   /// @inheritdoc IDCAStrategiesManagementHandler
-  function updateStrategyTokens(uint80 _strategyId, IDCAStrategies.ShareOfToken[] memory _tokens) external override {}
+  function updateStrategyTokens(uint80 _strategyId, IDCAStrategies.ShareOfToken[] memory _tokens) external override onlyStratOwner(_strategyId) {
```

Comment:
I think that `override` is not longer necessary since [0.8.8](https://blog.soliditylang.org/2021/09/27/solidity-0.8.8-release-announcement/)